app/views/characters/_list_item.haml

```diff
@@ -0,0 +1,30 @@
+- char_id, char_name, nickname, screenname, pb, user_id, username, user_deleted = character
+- if show_template
+  - template_id, template_name = character[-2..]
+- klass = cycle('even', 'odd')
+%tr
+  %td.padding-5{class: klass}
+    = link_to char_name, character_path(char_id)
+  - if local_assigns[:show_template]
```

Comment:
Since we already ate `local_assigns` from the above `show_template` reference, here too?

Reply:
Whoops! yup I'll fix that